ftp/BachJS/BWV1050/brand5-3/brand5-3-lys/header.ly

```diff
@@ -35,123 +37,100 @@
 		 My friends (music friends, school friends)
 		 My church (brethren in Christ)
 		 Friendly net pals (who helped me along)
-	 and in one or  many other aspects in my life
+	 and in one or many other aspects in my life
 
 	 I'd love to hear any comments. Email me at joshuakoo@myrealbox.com or zz85@users.sourceforge.net
 
 	 Joshua Koo
 
- %}
-#(ly:set-point-and-click 'line-column)
+         --------------
+         I updated this piece to LilyPond 2.18. After contacting Joshua Koo, we
+         agreed that I can contribute to this piece in addition to updating it.
+         The musical content is still what Joshua entered and I left the above
+         dedication and comments. Main change to the content was to make use of
+         the A-B-A structure of the piece and fix some typos where the two A
+         parts showed differences.
+
+         Joram Berger
+%}
+
 #(set-global-staff-size 18)
 
- \header {
-	 title = "Brandenberg Concerto No.5 (Keyboard Concencerto)"
-	 subtitle = "3rd Movement"
-	 composer = "J.S. Bach"
-	 meter = "Allegro"
-
-	%instrument = "Baroque Chamber"
-	%dedication = "Typesetted for my Friends"
-	%piece = ""
-	%head = ""
-	%footer = ""
-	%tagline = "joshuakoo@myrealbox.com. WIP for mutopia-project"
-
-	 mutopiatitle = "Brandenburg Concerto No. 5 in D Major"
-	 mutopiacomposer = "BachJS"
-	 mutopiaopus = "BWV 1050"
-	 mutopiainstrument = "Ensemble: Flute, Two Violins, Violas, 'Cello, Bass and Harpsichord"
-	 date = "1719?"
-	 source = "Dover From the Bach-Gesellschaft Edition"
-	 style = "Baroque"
-	 copyright = "Public Domain"
-	 maintainer = "Joshua Koo"
-	 maintainerEmail = "zz85@users.sourceforge.net"
-	 lastupdated = "2005/March/31"
-
-         footer = "Mutopia-2005/04/11-548"
-         tagline = "\\raisebox{10mm}{\\parbox{188mm}{\\quad\\small\\noindent " + \footer + " \\hspace{\\stretch{1}} This music is part of the Mutopia project: \\hspace{\\stretch{1}} \\texttt{http://www.MutopiaProject.org/}\\\\ \\makebox[188mm][c]{It has been typeset and placed in the public domain by " + \maintainer + ".} \\makebox[188mm][c]{Unrestricted modification and redistribution is permitted and encouraged---copy this music and share it!}}}"
+\header {
+  title = "Brandenburg Concerto No. 5 in D Major"
+  subtitle = "3rd Movement"
+  composer = "Johann Sebastian Bach"
+
+  mutopiatitle = "Brandenburg Concerto No. 5 in D Major"
+  mutopiacomposer = "BachJS"
+  mutopiaopus = "BWV 1050"
+  mutopiainstrument = "Ensemble: Flute, Two Violins, Violas, 'Cello, Bass and Harpsichord"
+  date = "1719?"
+  source = "Dover From the Bach-Gesellschaft Edition"
+  style = "Baroque"
+  copyright = ##f
+  maintainer = "Joshua Koo"
+  maintainerEmail = "zz85@users.sourceforge.net"
+  lastupdated = "2015-09-27"
+
+  footer = "Mutopia-2005/04/11-548"
+  thecopyright = "Creative Commons Attribution-ShareAlike 4.0"
```

Comment:
So `thecopyright` should also be `"Public Domain"` instead of `"Creative Commons Attribution-ShareAlike 4.0"`? Or should `thecopyright` be deleted as the field is now called `license`?